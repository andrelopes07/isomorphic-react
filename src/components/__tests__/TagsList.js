import React from 'react';
import renderer from 'react-test-renderer';
import TagsList from '../TagsList';

describe('The Tags List', () => {
    it('Should match snapshot', () => {
        const component = renderer
        .create(<TagsList tags={['html', 'css', 'javascript']} />)
        .toJSON();

        expect(component).toMatchSnapshot();
    });
});
