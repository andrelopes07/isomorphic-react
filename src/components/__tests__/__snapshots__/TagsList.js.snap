// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Tags List Should match snapshot 1`] = `
<div>
  <code>
    html
  </code>
  <code>
    css
  </code>
  <code>
    javascript
  </code>
</div>
`;
